BootCampV2/Core/Core.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BootCampV2.Homeworks.SecondWeek;
+using BootCampV2.Homeworks.ThirdWeek;
+using BootCampV2.Homeworks.ThirdWeek.Command;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BootCampV2
+{
+    public class Core
+    {
+        static public void Main(string[] args)
+        {
+            string path = @".\User.txt";
+
+            UserFileManager userFileManager = new UserFileManager(path);
+
+            ICommand[] commands = {
+                new LoginCommand(userFileManager),
+                new RegisterCommand(userFileManager),
+                new ExitCommand()
+            };
+
+            CommandManager commandManager = new CommandManager(commands);
```

Comment:
A blank line in between is messing up with the readability. Consider putting related code close and unrelated one further away from the lines.